src/lib.rs

```diff
@@ -394,8 +450,10 @@ Resources:
 
     const PARSING_POSITIVE: &str = include_str!("../tests/siwe/test/parsing_positive.json");
     const PARSING_NEGATIVE: &str = include_str!("../tests/siwe/test/parsing_negative.json");
-    const VALIDATION_POSITIVE: &str = include_str!("../tests/siwe/test/validation_positive.json");
-    const VALIDATION_NEGATIVE: &str = include_str!("../tests/siwe/test/validation_negative.json");
+    const VERIFICATION_POSITIVE: &str =
+        include_str!("../tests/siwe/test/verification_positive.json");
+    const VERIFICATION_NEGATIVE: &str =
+        include_str!("../tests/siwe/test/verification_negative.json");
```

Comment:
can we rename the test cases as well to show the new naming scheme? i saw that some of the test cases are still called validation although they do eip-191 verifcation internally.